src/utils.cpp

```diff
@@ -1296,4 +1310,29 @@ std::pair<std::string, std::string> split_symbol_module(
                          symbol.length() - idx - strlen(" []")) };
 }
 
+// Usually the /sys/kernel/debug/kprobes/blacklist file.
+// Format example:
+// 0xffffffff85201511-0xffffffff8520152f	first_nmi
+// 0xffffffffc17e9373-0xffffffffc17e94ff	vmx_vmexit [kvm_intel]
+// The outputs are:
+// { "0xffffffff85201511-0xffffffff8520152f", "first_nmi", "" }
+// { "0xffffffffc17e9373-0xffffffffc17e94ff", "vmx_vmexit", "kvm_intel" }
+std::tuple<std::string, std::string, std::string> split_addrrange_symbol_module(
+    const std::string &symbol)
```

Comment:
Do we really need such a complicated implementation when we practically need only the function name?

Reply:
Here I want to be consistent with `split_symbol_module()`, the blacklist contains ('func', 'module') information, which may be used in subsequent development. Moreover, this function does not require additional maintenance costs.

Reply:
Ok, the module name will probably be used eventually, so let's keep it as it is.